src/include/btmem.h

```diff
@@ -938,13 +938,23 @@ struct __wt_ref_hist {
 
 /*
  * WT_PREFETCH --
- *	Queue entry for pages queued for pre-fetch.
+ *	Pre-fetch structure containing useful information for pre-fetch.
  */
 struct __wt_prefetch {
+    WT_REF *prefetch_prev_ref;
+    uint64_t prefetch_disk_read_count; /* Sequential cache requests that caused a leaf read */
+    uint64_t prefetch_skipped_with_parent;
+};
```

Comment:
Can we define this data structure in the `session.h` file instead? We should also delete the variables in the WT_SESSION_IMPL structure and replace it with the new data structure.

Reply:
Yes of course, sorry I missed adding that.